src/mlpack/methods/ann/layer/max_pooling.hpp

```diff
@@ -179,25 +179,25 @@ class MaxPooling
                         arma::Mat<eT>& output,
                         arma::Mat<eT>& poolingIndices)
   {
+    const size_t rStep = kernelWidth;
+    const size_t cStep = kernelHeight;
```

Comment:
Hey @kartikdutt18,
I was just going through the changes. I was having a doubt. Is it necessary to declare these two variables over here? I think we can avoid them. let me know your thoughts on it. Maybe I missed something.

Reply:
Sure I'll make the make in the next PR that open. Thanks for pointing them out.